### PR TITLE
fix post http authorize

### PR DESCRIPTION
**Pull request by liandeliang**
add kucoin trade order error code
add CreateOrderByString() to fix precise problem

### CODE REVIEW

File: kucoin.go [hunk 1]
```diff
@@ -365,6 +365,44 @@ func (b *Kucoin) CreateOrder(symbol, side string, price, amount float64) (orderO
...
+	if !rawRes.Success {
+		err = errors.New(string(r))
+		return
+	}
+	orderOid = rawRes.Data.OrderOid
+	return
+}
+
+// CreateOrderByString is used to create order at Kucoin along with other meta data. This ByString version is fix precise problem.
+func (b *Kucoin) CreateOrderByString(symbol, side string, price, amount string) (orderOid string, err error) {
```
Comment (eeonevision) [inline kucoin.go:380]: Nice! Remove "string" type after "side" arg. declaration, it just unneeded in case of all string args.

File: kucoin.go [hunk 2]
```diff
@@ -365,6 +365,44 @@ func (b *Kucoin) CreateOrder(symbol, side string, price, amount float64) (orderO
...
+	}
+	if !rawRes.Success {
+		err = errors.New(string(r))
+		return
+	}
+	orderOid = rawRes.Data.OrderOid
+	return
+}
+
+// CreateOrderByString is used to create order at Kucoin along with other meta data. This ByString version is fix precise problem.
```
Comment (eeonevision) [inline kucoin.go:379]: Move comment after '.' at second line. This improves readability.

**Review by eeonevision — APPROVED** [review]

File: kucoin.go (hunk 1 above)
Comment (liandeliang) [reply kucoin.go:380]: ok, what should i do? i am a submission beginner.

Comment (eeonevision) [reply kucoin.go:380]: Change method arguments from:
```code 
symbol, side string, price, amount string
```
to:
```code
symbol, side, price, amount string
```

Comment (eeonevision) [reply kucoin.go:380]: And after, new commit with fixed code. It will automatically update pull request.

**Review by eeonevision — APPROVED** [review]
Fix and it will be approved

**Pull request merged**